libs/agno/agno/workflow/v2/workflow.py

```diff
@@ -1345,6 +1464,7 @@ async def arun(
         stream: Literal[False] = False,
         stream_intermediate_steps: Optional[bool] = None,
         background: Optional[bool] = False,
+        websocket: Optional[WebSocket] = None,
```

Comment:
Hmm should we not instantiate the agent with this? Otherwise it breaks for concurrent runs?